packages/test-base/src/androidTest/kotlin/io/realm/kotlin/test/shared/MutableRealmTests.kt

```diff
@@ -675,4 +678,31 @@ class MutableRealmTests {
             }
         }
     }
+
+    @Test
+    fun deleteAll() {
```

Comment:
We are missing tests for `delete(className)` both in the success case and if you provide a classname that isn't part of the schema both for typed and dynamic realms.